### PR TITLE
Fix gNMI-1.2:  Benchmarking: Full Configuration Replace  test failure in EOS

### PR DESCRIPTION
**Pull request by prakashbadri-arista**
The test was failing for the following reasons
- "network-instances/network-instance/protocols/protocol/isis/global/config/instance" ( Isis global instance configuration ) was not set
- LSP refresh interval not set when LSP lifetime interval is set. In google B4 deployment ( DVT test config ), I do see lsp refresh interval being set.
- Unsupported isis auth OC paths were being set.
- Unsupported per interface afi/level specific config was being enabled

After fixing the above issues, the test is passing -
-

### CODE REVIEW

File: feature/experimental/system/gnmi/benchmarking/ate_tests/internal/setup/setup.go [hunk 1]
```diff
@@ -155,6 +155,9 @@ func BuildBenchmarkingConfig(t *testing.T) *oc.Root {
 	isis := prot.GetOrCreateIsis()
 
 	globalISIS := isis.GetOrCreateGlobal()
+	if !deviations.ISISInstanceEnabledNotRequired(dut) {
+		globalISIS.Instance = ygot.String(ISISInstance)
+	}
```
[the remark this answers is not in the document]
Comment (Ankur19) [reply feature/experimental/system/gnmi/benchmarking/ate_tests/internal/setup/setup.go:160]: Also need to update `metadata.proto` with this deviation applied for Cisco and Nokia

**Review by Ankur19 — CHANGES_REQUESTED** [review]

File: feature/experimental/system/gnmi/benchmarking/ate_tests/internal/setup/setup.go [hunk 2]
```diff
@@ -165,6 +168,7 @@ func BuildBenchmarkingConfig(t *testing.T) *oc.Root {
 	lspBit.SetBit = ygot.Bool(false)
 	isisTimers := globalISIS.GetOrCreateTimers()
 	isisTimers.LspLifetimeInterval = ygot.Uint16(600)
+	isisTimers.LspRefreshInterval = ygot.Uint16(100)
```
[the remark this answers is not in the document]
Comment (LimeHat) [reply feature/experimental/system/gnmi/benchmarking/ate_tests/internal/setup/setup.go:171]: @Ankur19 do we have to use 100 in the first place?
Based in conversation in b/289374850 I thought it wasn't needed

File: feature/experimental/system/gnmi/benchmarking/ate_tests/internal/setup/setup.go [hunk 3]
```diff
@@ -244,8 +247,6 @@ func BuildBenchmarkingConfig(t *testing.T) *oc.Root {
 		// Configure ISIS AfiSafi enable flag at the global level
 		if deviations.MissingIsisInterfaceAfiSafiEnable(dut) {
 			isisIntf.GetOrCreateAf(oc.IsisTypes_AFI_TYPE_IPV4, oc.IsisTypes_SAFI_TYPE_UNICAST).Enabled = ygot.Bool(true)
-		} else {
-			isisIntfLevelAfi.Enabled = ygot.Bool(true)
```
Comment (LimeHat) [inline feature/experimental/system/gnmi/benchmarking/ate_tests/internal/setup/setup.go:248]: Isn't the above deviation (`deviations.MissingIsisInterfaceAfiSafiEnable(dut)`) supposed to be used when this line is not supported?

**Comment by LimeHat** [comment]
Thanks for the changes @Ankur19, looks good

File: feature/experimental/system/gnmi/benchmarking/ate_tests/internal/setup/setup.go (hunk 3 above)
Comment (Ankur19) [reply feature/experimental/system/gnmi/benchmarking/ate_tests/internal/setup/setup.go:248]: Done!

**Review by Ankur19 — APPROVED** [review]
LGTM!

**Review by Ankur19 — DISMISSED** [review]
LGTM!

**Review by Ankur19 — APPROVED** [review]

**Pull request merged**